common/scripted_effects/00_scripted_effects.txt

```diff
@@ -5230,6 +5230,11 @@ remove_symptoms_effect = {
 		limit = { trait = malaise}
```

Comment:
No localization.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79054371-4b8fc880-7c55-11ea-9f75-4fa04f7d9fc3.png)

</details>